composer/cli/launcher.py

```diff
@@ -166,11 +182,6 @@ def monitor_processes(processes: Set[subprocess.Popen]):
 
 
 def cleanup_processes(processes: Set[subprocess.Popen]):
-    if len(processes) == 0:
```

Comment:
Curious to know the reason we no longer need these cleanup checks

Reply:
The launched processes, not the launcher, should call the destroy process group. But since we use the same process group for the entire duration of the python process (regardless of how many times .fit() is called), we do not destroy it anymore.